web/src/use/use-user.ts

```diff
@@ -25,6 +25,8 @@ export function useUser(
     isErrored: false,
   })
 
+  const isSystemAdmin = computed(() => state.user?.roleTypes.includes(RoleTypes.SYSTEM_ADMIN))
```

Comment:
Is this being used anywhere? If not, I don't think it's likely to be used in the future either.